src/jit/compiler.h

```diff
@@ -2274,7 +2274,11 @@ public :
 #endif // !LEGACY_BACKEND
 
     void                lvaAssignVirtualFrameOffsetsToArgs();
-    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs);
+    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs
+#ifdef UNIX_AMD64_ABI
+                                                         , int& callerArgOffset
```

Comment:
Is this argument in/out? We prefer int* to int&. Our conventions say that reference arguments should be 'const'.